code/mob/living/critter/ai/roach.dm

```diff
@@ -0,0 +1,8 @@
+/datum/aiHolder/roach
+	New()
+		..()
+		default_task = get_instance(/datum/aiTask/prioritizer/critter/roach, list(src))
```

Comment:
```suggestion
		default_task = get_instance(/datum/aiTask/timed/wander, list(src))
```
and delete the prioritzer. You don't need a prioritizer task for just one ai task.

Reply:
Gonezo, keeping it in mind.